uint32 id
uint8 state
geometry_msgs/PoseStamped pose

uint8 ON_GROUND = 0
uint8 PICKED_UP = 1
uint8 DELIVERED = 2
